src/assets/images/index.js

```diff
@@ -0,0 +1,8 @@
+import bags from '../images/bags.jpg';
+import footWear from '../images/foot wear.jpg';
+import tshirt from '../images/tshirt.jpg';
+import uiLogo from '../images/ui-logo.jpg';
+import watch from '../images/watch.jpg';
+import westernWear from '../images/western-wear.jpg';
+
+export {bags,footWear,tshirt,uiLogo,watch,westernWear};
```

Comment:
Same here. Add whitespace between commas and the brackets. Use a formatter like prettier and setup your editor to format on save.

Reply:
done.